File: submissions/sm_035_yashwanth/week_3/day_3/session_2/count_consonants.js
var string="yashwanth"
var vowels=['b','c','d','f','g','h','j','k','l','m','n','p','q','r','s','t','v','w','x','y','z']
count=0
  for(i=0;i<string.length;i++){
      for(j=0;j<vowels.length;j++){
       if(string[i]==vowels[j]){
        count += 1
       }
      }
  }


console.log(count)

File: submissions/sm_009_chinmay/week_13/day_4/part_1/count_consonants.py
# Print the count of all the consonants in the given list of strings

vowels = ['a','e','i','o','u']
list_of_strings = ['a','b','c','d','e','f','g','h','i', 'o', 'u'] #consonants count should be 6
cnt = len(list_of_strings)

for i in range(len(list_of_strings)):
    for j in range(len(vowels)):
        if list_of_strings[i] == vowels[j]:
            cnt = cnt - 1

print(cnt)